src/engraving/libmscore/score.cpp

```diff
@@ -5394,33 +5394,26 @@ void Score::changeSelectedNotesVoice(voice_idx_t voice)
     setLayoutAll();
 }
 
-std::set<ID> Score::partIdsFromRange(const track_idx_t trackFrom, const track_idx_t trackTo) const
-{
-    std::set<ID> result;
-
-    for (const Part* part : m_score->parts()) {
-        if (trackTo < part->startTrack() || trackFrom >= part->endTrack()) {
-            continue;
-        }
-
-        result.insert(part->id());
-    }
-
-    return result;
-}
-
-std::set<staff_idx_t> Score::staffIdsFromRange(const track_idx_t trackFrom, const track_idx_t trackTo) const
+std::set<staff_idx_t> Score::staffIdsFromRange(const track_idx_t trackFrom, const track_idx_t trackTo, StaffAccepted staffAccepted) const
```

Comment:
Just realised one thing: this method is called <code>staff<I><b>Ids</b></I>fromRange</code>, but it returns staff _**indexes**_. No big deal right now, but this is prone to lead to confusion, because staves _also_ have IDs, that don't necessarily correspond to staff indexes.